pkg/thing/mocks/publisher.go

```diff
@@ -0,0 +1,29 @@
+package mocks
+
+import (
+	"github.com/CESARBR/knot-babeltower/pkg/network"
+	"github.com/CESARBR/knot-babeltower/pkg/thing/entities"
+	"github.com/stretchr/testify/mock"
+)
+
+type FakePublisher struct {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported type FakePublisher should have comment or be unexported](https://app.codacy.com/manual/joaoaneto/knot-babeltower/pullRequest?prid=4853780)